test/checks/landmarks/landmark-is-unique.js

```diff
@@ -0,0 +1,44 @@
+describe('landmark-is-unique', function() {
+	'use strict';
+
+	var checkContext = new axe.testUtils.MockCheckContext();
+	let fixture;
```

Comment:
As above, we can't use ES2015 features in tests. Please use `var` instead of `let` and `const`.

Reply:
Done.